### PR TITLE
Fixes some issues

### PR DESCRIPTION
**Pull request by Naramsim**
https://github.com/PokeAPI/pokeapi/issues/703
https://github.com/PokeAPI/pokeapi/issues/700
and another one I cannot find anymore

### CODE REVIEW

**Comment by C-Garza** [comment]
[Here is the Azure flute issue for reference.](https://github.com/PokeAPI/api-data/issues/99)

File: data/v2/csv/pokemon_species.csv [hunk 1]
```diff
@@ -808,87 +808,87 @@ id,identifier,generation_id,evolves_from_species_id,evolution_chain_id,color_id,
...
+884,duraludon,8,,465,9,6,,4,45,50,0,30,0,2,0,0,0,884,
+885,dreepy,8,,466,5,2,,4,45,50,0,40,0,1,0,0,0,885,
+886,drakloak,8,885,466,5,4,,4,45,50,0,40,0,1,0,0,0,886,
+887,dragapult,8,886,466,5,6,,4,45,50,0,40,0,1,0,0,0,887,
+888,zacian,8,,467,2,8,,-1,10,0,0,120,0,1,0,1,0,888,
+889,zamazenta,8,,468,8,8,,-1,10,0,0,120,0,1,0,1,0,889,
+890,eternatus,8,,469,7,9,,-1,255,0,0,120,0,1,0,1,0,890,
 891,kubfu,8,,470,4,,,1,3,50,0,120,0,1,0,1,0,891,
 892,urshifu,8,891,470,4,,,1,3,50,0,120,0,1,0,1,0,892,
 893,zarude,8,,471,5,,,-1,3,0,0,120,0,1,0,0,1,893,
```
Comment (C-Garza) [inline data/v2/csv/pokemon_species.csv]: Looks like 891-899 still have their shapes missing.
```
891: 6,
892: 12,
893: 6,
894: 12,
895: 12,
896: 8,
897: 8,
898: 12
```

**Review by C-Garza — APPROVED** [review]
Looks good to me.

**Comment by pokeapi-machine-user** [comment]
A [PokeAPI/api-data](https://github.com/PokeAPI/api-data) refresh has started. In 45 minutes the staging branch of [PokeAPI/api-data](https://github.com/PokeAPI/api-data/tree/staging) will be pushed with the new generated data. <br><br> The staging branch will be deployed in our [staging environment](https://staging.pokeapi.co/api/v2/) and you will be able to review the entire API. <br><br> A Pull Request ([master](https://github.com/PokeAPI/api-data/tree/master)<-[staging](https://github.com/PokeAPI/api-data/tree/staging)) will be also created at [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) and assigned to the PokeAPI Core team to be reviewed. If approved and merged new data will soon be available worldwide at [pokeapi.co](https://pokeapi.co/api/v2/).

**Comment by pokeapi-machine-user** [comment]
The updater script has finished its job and has now opened a Pull Request towards [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) with the updated data. <br><br> You can see the Pull Request deployed at our [staging environment](https://staging.pokeapi.co/api/v2/) when [CircleCI deploy](https://app.circleci.com/pipelines/github/PokeAPI/deploy) will be finished (_check the started time of the last build_).

**Pull request merged**